app/components/exp-lookit-iframe/doc.rst

```diff
@@ -0,0 +1,68 @@
+.. _exp-lookit-iframe:
+
+exp-lookit-iframe
+==============================================
+
+Overview
+------------------
+
+A frame to display external websites, such as surveys, to the participant. 
+
+What it looks like
+~~~~~~~~~~~~~~~~~~
+
+.. image:: /../images/Exp-lookit-iframe.png
+    :alt: Example screenshot from exp-lookit-iframe frame
+
+More general functionality
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Below is information specific to this particular frame. There may also be available parameters, events recorded,
+and data collected that come from the following more general sources:
+
+- the :ref:`base frame<base frame>` (things all frames do)
+
+Examples
+----------------
+
+.. code:: javascript
+
+    "study-survey": {
+        "kind": "exp-lookit-iframe",
+        "iframeSrc": "https://mit.co1.qualtrics.com/jfe/form/SV_do6kyKJkwl19USW",
+        "iframeHeight": "700px",
+        "iframeWidth": "100%",
+        "optionalText": "Message to the participant."
+    }
+
+Parameters
+----------------
+
+iframeSrc [String]
+    Page that should be shown in IFrame.  Additionally, we update the query string with child's hash id using the key value 'child'
```

Comment:
proposed instructions update: 

iframeSrc [String]
The external URL that should be shown in the iframe. The link will be automatically updated to include two pieces of information as URL query parameters: the hashed child ID (‘child’) and the response ID (‘response’). This will allow you to link the study responses and child’s Lookit account without having to ask the family to enter additional information. See this page for information on how to use these query strings: https://lookit.readthedocs.io/en/develop/researchers-set-study-fields.html#study-url-external-studies

Also, (when) do we validate this as a legal URL?

Reply:
(If it's possible to hide that ugly link behind the [this page] text then by all means do so, I just forgot my markdown for a moment)

Reply:
Validation is done in function to add the params.